src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,263 @@
+                        - name: MYSQLMostViewedProductsDaily
+                        run: SELECT SUM(views_num) as 'viewCount', product_name FROM report_viewed_product_aggregated_daily GROUP BY product_name ORDER BY viewCount DESC;
+                        - name: MYSQLConversionRate
+                        run: WITH cte1 AS (SELECT soi.product_id AS 'SalesProductId', soi.name AS 'ProductName', COUNT(soi.order_id) AS 'totalOrders' FROM `sales_order_item` soi group by soi.product_id, soi.name), cte2 AS (SELECT viewed.product_id AS 'productId', COUNT(viewed.product_id) AS 'totalViews' FROM `report_viewed_product_index` viewed group by viewed.product_id) SELECT SalesProductId as 'productID', ProductName, totalViews,totalOrders,totalOrders/totalViews *100 as 'conversionRateInPercent' FROM cte1 JOIN cte2 WHERE cte1.SalesProductId = cte2.ProductId;
+                        - name: MYSQLTopTenSalesOrdersByValue
+                        run: SELECT entity_id AS 'orderId', total_item_count AS 'orderQuantity', created_at AS 'createdDate', grand_total AS 'grandTotal' FROM sales_order ORDER BY grand_total DESC LIMIT 10;
+                        - name: MYSQLOrdersAverage
+                        run: SELECT AVG(order_count) AS 'AverageNumberOfOrders', Year AS 'Year' FROM (SELECT YEAR(created_at) AS 'Year', COUNT(DISTINCT order_id) AS order_count FROM sales_order_item GROUP BY MONTH(created_at), YEAR(created_at)) subquery GROUP BY Year;
+                        - name: MYSQLProductsTotal
+                        run: SELECT count(*) AS 'Total Products' FROM catalog_product_entity_varchar WHERE attribute_id = 73;
+                        custom_attributes: # can apply additional at a nested level
+                            attype: blocking-query
+                            host: CustomMagentoMySQL
+                ```
+
+        3. [Restart the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/).
+    
+    </Step>
+
+<Step>
+
+    ## Install the Magento business insights dashboard [#find-data]
+
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards**.
+        2. In the search bar, type "Magento business insights". The dashboard should appear. 
+        3. Click on it to install.
+
+Your Magento dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For information on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+        </Step>
+</Steps>
+
+## Query your Magento data
+
+Interested in querying your Magento app data? Here's an example NRQL query checking the total registered customers on your Magento platform:
```

Comment:
for these kinds of integrations I wondered if a good boilerplate thing to say is something like "This integration reports data from all the agents and integrations involved in its install. For details on the data reported by those tools, see the docs for those tools. Here's an example of..."